self-core-impl/src/main/java/com/selfxdsd/core/GithubInvitation.java

```diff
@@ -66,7 +65,7 @@
     /**
      * Parent Github.
      */
-    private final Github github;
+    private final Provider github;
```

Comment:
@criske This class represents a Github Invitation, why does this attribute have to be a ``Provider``?

Reply:
@amihaiemil I've needed to verify in tests that `Stars#add` is called or not (a concrecte impl of `Provider` wouldn't let me as it can't be mocked). But I can change it back and check it with `MockJsonResources` if stars endpoint is called.

Reply:
@criske yes please, the MockJsonResources version would be better. I don't want to have workarounds in the code just because of the tests :D